README.md

```diff
@@ -1,3 +1,10 @@
+# dslibAlfy-vesion
```

Comment:
Please name it dslib thread-safe version (we don't put names in code other than Contributor section).